x/tokenfactory/keeper/keeper_test.go

```diff
@@ -38,32 +38,45 @@ func (suite *KeeperTestSuite) Setup() {
 		Ctx:             suite.ChainA.GetContext(),
 	}
 	suite.TestAccs = CreateRandomAccounts(3)
-	fmt.Printf("Setup: %s\n", suite.TestAccs)
 
 	suite.SetupTokenFactory()
 
 	suite.queryClient = types.NewQueryClient(suite.QueryHelper)
 
 	tokeFactoryKeeper := suite.GetNeutronZoneApp(suite.ChainA).TokenFactoryKeeper
+	tokeFactoryKeeper.SetParams(suite.ChainA.GetContext(), types.NewParams(
+		sdktypes.NewCoins(sdktypes.NewInt64Coin(types.DefaultNeutronDenom, 1_000_000)),
+		"neutron1vguuxez2h5ekltfj9gjd62fs5k4rl2zy5hfrncasykzw08rezpfsd2rhm7",
```

Comment:
please move the addr to a const and describe it. I can see it's used in different places across the code